developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -78,59 +73,57 @@ At creation, an [Aptos account](https://github.com/aptos-labs/aptos-core/blob/88
 
 ### Creating an account address
 
-Account addresses are defined at creation time as a one-way function from the public key(s) and signature algorithm used for authentication for the account.
+[Account addresses](../concepts/basics-accounts.md#account-address) are defined at creation time as a one-way function from the public key(s) and signature algorithm used for authentication for the account.
 
 :::tip Read more
-This is covered in depth in the [Accounts](https://aptos.dev/concepts/basics-accounts/) documentation and demonstrated in the [Typescript SDK](https://github.com/aptos-labs/aptos-core/blob/9b85d41ed8ef4a61a9cd64f9de511654fcc02024/ecosystem/typescript/sdk/src/aptos_account.ts#L66) and [Python SDK](https://github.com/aptos-labs/aptos-core/blob/9b85d41ed8ef4a61a9cd64f9de511654fcc02024/ecosystem/python/sdk/aptos_sdk/account_address.py#L43). Note that currently these SDKs only demonstrate how to generate an address from an Ed25519 single signer.
+This is covered in depth in the [Accounts](../concepts/basics-accounts.md) documentation and demonstrated in the [Typescript SDK](https://github.com/aptos-labs/aptos-core/blob/9b85d41ed8ef4a61a9cd64f9de511654fcc02024/ecosystem/typescript/sdk/src/aptos_account.ts#L66) and [Python SDK](https://github.com/aptos-labs/aptos-core/blob/9b85d41ed8ef4a61a9cd64f9de511654fcc02024/ecosystem/python/sdk/aptos_sdk/account_address.py#L43). Note that currently these SDKs demonstrate only how to generate an address from an Ed25519 single signer.
 :::
 
 ### Rotating the keys
 
-An Account on Aptos has the ability to rotate keys so that potentially compromised keys cannot be used to access the accounts. Keys can be rotated via [`account::rotate_authentication_key`](https://github.com/aptos-labs/aptos-core/blob/60751b5ed44984178c7163933da3d1b18ad80388/aptos-move/framework/aptos-framework/sources/account.move#L183) function.
-
-:::tip Read more
-See more in [Account address](/concepts/basics-accounts#account-address).
-:::
+An Account on Aptos has the ability to rotate keys so that potentially compromised keys cannot be used to access the accounts. Keys can be rotated via the [`account::rotate_authentication_key`](https://github.com/aptos-labs/aptos-core/blob/60751b5ed44984178c7163933da3d1b18ad80388/aptos-move/framework/aptos-framework/sources/account.move#L183) function.
 
 Refreshing the keys is generally regarded as good hygiene in the security field. However, this presents a challenge for system integrators who are used to using a mnemonic to represent both a private key and its associated account. To simplify this for the system integrators, Aptos will provide an on-chain mapping, before the launch of the mainnet. The on-chain data maps an effective account address as defined by the current mnemonic to the actual account address.
 
+TODO: When will we provide an on-chain mapping, before the launch of the mainnet?
```

Comment:
On chain mapping?  it already exists, you can remove this line.

See `aptos account lookup-address`

Reply:
General comment: Let's remove all TODO appearances from the rendered docs by commenting them out in the Markdown source.

Reply:
> General comment: Let's remove all TODO appearances from the rendered docs by commenting them out in the Markdown source.

Done. That was the lone one. Greg, note i updated the paragraph preceding the TODO with your input.